k8s/apis/k8s/v1alpha1/vineyardd_types.go

```diff
@@ -167,6 +167,35 @@ type VineyardContainerConfig struct {
 	Env []corev1.EnvVar `json:"env,omitempty"`
 }
 
+// OperationImageConfig holds all image configuration about operations(backup, recover,
+// local assembly, distributed assembly, repartition)
+type OperationImageConfig struct {
```

Comment:
`PluginConfig` or `DriverConfig` would be a better name, as in our design we think they are pluggable drivers.

Reply:
Done and `Artifact Hub repository` updated too.